lzo.spec

```diff
@@ -54,46 +39,25 @@ This package contains development files needed for lzo.
     --enable-shared \
     CFLAGS="`echo $CFLAGS | sed 's/-O2//g'`"
```

Comment:
I wonder if we could drop this line now. It seems to generate some warnings like this one:
```
/usr/include/features.h:382:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
 #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
    ^~~~~~~
```

Reply:
Makes sense. We shouldn't be doing this sort of adjustments anyway if there's no clear need.